<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated with glade 3.20.0 -->
<interface>
  <requires lib="gtk+" version="3.18"/>
  <template class="NautilusActionBar" parent="GtkFrame">
    <property name="visible">True</property>
    <property name="can_focus">False</property>
    <property name="vexpand">False</property>
    <property name="label_xalign">0</property>
    <child>
      <object class="GtkStack" id="stack">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="valign">center</property>
        <property name="hexpand">True</property>
        <property name="border_width">6</property>
        <property name="hhomogeneous">False</property>
        <property name="transition_type">crossfade</property>
        <child>
          <object class="GtkBox">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="valign">center</property>
            <property name="hexpand">True</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkButton" id="new_folder_0_button">
                <property name="label" translatable="yes">New Folderâ€¦</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.new-folder</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="paste_button">
                <property name="label" translatable="yes">Paste</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.paste</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="select_all_button">
                <property name="label" translatable="yes">Select All</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.select-all</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">2</property>
              </packing>
            </child>
            <child>
              <object class="GtkSeparator" id="no_selection_separator">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">3</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="bookmark_button">
                <property name="label" translatable="yes">Bookmark</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">4</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="properties_0_button">
                <property name="label" translatable="yes">Properties</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.properties</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">5</property>
              </packing>
            </child>
            <child>
              <object class="GtkMenuButton" id="no_selection_overflow_button">
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="direction">up</property>
                <property name="popover">no_selection_popover</property>
                <child>
                  <object class="GtkImage">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="icon_name">view-more-symbolic</property>
                  </object>
                </child>
                <style>
                  <class name="image-button"/>
                </style>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">6</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="no_selection_folder_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="margin_right">12</property>
                <property name="ellipsize">middle</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="pack_type">end</property>
                <property name="position">7</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="name">no-selection</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="valign">center</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkSpinner">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="active">True</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">0</property>
              </packing>
            </child>
            <child>
              <object class="GtkLabel" id="loading_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="label" translatable="yes">Loading</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="position">1</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton">
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.stop-loading</property>
                <child>
                  <object class="GtkImage">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="icon_name">process-stop-symbolic</property>
                  </object>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="pack_type">end</property>
                <property name="position">2</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="name">loading</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkBox">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
            <property name="spacing">6</property>
            <child>
              <object class="GtkBox" id="open_file_box">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <style>
                  <class name="linked" />
                </style>
                <child>
                  <object class="GtkButton" id="default_app_button">
                    <property name="label" translatable="yes">Open</property>
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="receives_default">True</property>
                    <property name="action_name">view.open-with-default-application</property>
                  </object>
                </child>
                <child>
                  <object class="GtkButton" id="other_apps_button">
                    <property name="label" translatable="yes">Open</property>
                    <property name="visible">True</property>
                    <property name="can_focus">True</property>
                    <property name="receives_default">True</property>
                    <property name="action_name">view.open-with-default-application</property>
                  </object>
                </child>
              </object>
            </child>
            <child>
              <object class="GtkButton" id="open_folders_button">
                <property name="label" translatable="yes">Open</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.open-with-default-application</property>
              </object>
            </child>
            <child>
              <object class="GtkButton" id="move_folders_button">
                <property name="label" translatable="yes">Move Toâ€¦</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.move-to</property>
              </object>
            </child>
            <child>
              <object class="GtkButton" id="copy_folders_button">
                <property name="label" translatable="yes">Copy Toâ€¦</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.copy-to</property>
              </object>
            </child>
            <child>
              <object class="GtkButton" id="rename_folders_button">
                <property name="label" translatable="yes">Renameâ€¦</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.rename</property>
              </object>
            </child>
            <child>
              <object class="GtkButton" id="move_trash_folders_button">
                <property name="label" translatable="yes">Move to Trash</property>
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="action_name">view.move-to-trash</property>
              </object>
            </child>
            <child>
              <object class="GtkMenuButton" id="folders_overflow_button">
                <property name="visible">True</property>
                <property name="can_focus">True</property>
                <property name="receives_default">True</property>
                <property name="direction">up</property>
                <property name="popover">folders_popover</property>
                <child>
                  <object class="GtkImage">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="icon_name">view-more-symbolic</property>
                  </object>
                </child>
                <style>
                  <class name="image-button"/>
                </style>
              </object>
            </child>
            <child>
              <object class="GtkLabel" id="folders_only_folder_label">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="ellipsize">middle</property>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="pack_type">end</property>
                <property name="position">6</property>
              </packing>
            </child>
            <child>
              <object class="GtkButton" id="clear_selection_0_button">
                <property name="visible">True</property>
                <property name="can_focus">False</property>
                <property name="tooltip-text" translatable="yes">Clear the selection</property>
                <style>
                  <class name="image-button" />
                  <class name="flat" />
                  <class name="circular" />
                </style>
                <child>
                  <object class="GtkImage">
                    <property name="visible">True</property>
                    <property name="can_focus">False</property>
                    <property name="icon_name">window-close-symbolic</property>
                  </object>
                </child>
              </object>
              <packing>
                <property name="expand">False</property>
                <property name="fill">True</property>
                <property name="pack_type">end</property>
                <property name="position">7</property>
              </packing>
            </child>
          </object>
          <packing>
            <property name="name">files-folders</property>
            <property name="position">2</property>
          </packing>
        </child>
      </object>
    </child>
  </template>
  <object class="GtkPopoverMenu" id="no_selection_popover">
    <property name="can_focus">False</property>
    <child>
      <object class="GtkBox">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="border_width">6</property>
        <property name="orientation">vertical</property>
        <child>
          <object class="GtkModelButton">
            <property name="visible" bind-source="new_folder_0_button" bind-property="visible" bind-flags="default|invert-boolean" />
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.new-folder</property>
            <property name="text" translatable="yes">New Folderâ€¦</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible" bind-source="paste_button" bind-property="visible" bind-flags="default|invert-boolean" />
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.paste</property>
            <property name="text" translatable="yes">Paste</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible" bind-source="select_all_button" bind-property="visible" bind-flags="default|invert-boolean" />
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.select-all</property>
            <property name="text" translatable="yes">Select All</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">2</property>
          </packing>
        </child>
        <child>
          <object class="GtkSeparator">
            <property name="visible" bind-source="select_all_button" bind-property="visible" bind-flags="default|invert-boolean" />
            <property name="can_focus">False</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">3</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible" bind-source="bookmark_button" bind-property="visible" bind-flags="default|invert-boolean" />
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.bookmark</property>
            <property name="text" translatable="yes">Bookmark</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">4</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.properties</property>
            <property name="text" translatable="yes">Properties</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">5</property>
          </packing>
        </child>
      </object>
      <packing>
        <property name="submenu">submenu0</property>
        <property name="position">1</property>
      </packing>
    </child>
  </object>
  <object class="GtkPopoverMenu" id="folders_popover">
    <property name="can_focus">False</property>
    <property name="relative_to">folders_overflow_button</property>
    <child>
      <object class="GtkBox">
        <property name="visible">True</property>
        <property name="can_focus">False</property>
        <property name="border_width">6</property>
        <property name="orientation">vertical</property>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.open-with-default-application</property>
            <property name="text" translatable="yes">Open</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">0</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.move-to</property>
            <property name="text" translatable="yes">Move Toâ€¦</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">1</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.copy-to</property>
            <property name="text" translatable="yes">Copy Toâ€¦</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">2</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.rename</property>
            <property name="text" translatable="yes">Renameâ€¦</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">3</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.move-to-trash</property>
            <property name="text" translatable="yes">Move to Trash</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">4</property>
          </packing>
        </child>
        <child>
          <object class="GtkSeparator">
            <property name="visible">True</property>
            <property name="can_focus">False</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">5</property>
          </packing>
        </child>
        <child>
          <object class="GtkModelButton">
            <property name="visible">True</property>
            <property name="can_focus">True</property>
            <property name="receives_default">True</property>
            <property name="action_name">view.properties</property>
            <property name="text" translatable="yes">Properties</property>
          </object>
          <packing>
            <property name="expand">False</property>
            <property name="fill">True</property>
            <property name="position">6</property>
          </packing>
        </child>
      </object>
      <packing>
        <property name="submenu">submenu0</property>
        <property name="position">1</property>
      </packing>
    </child>
  </object>
  <object class="GtkSizeGroup" id="files_folders_size_group">
    <widgets>
      <widget name="open_folders_button"/>
      <widget name="move_folders_button"/>
      <widget name="copy_folders_button"/>
      <widget name="rename_folders_button"/>
      <widget name="move_trash_folders_button"/>
    </widgets>
  </object>
  <object class="GtkSizeGroup" id="no_selection_size_group">
    <widgets>
      <widget name="new_folder_0_button"/>
      <widget name="paste_button"/>
      <widget name="select_all_button"/>
      <widget name="bookmark_button"/>
      <widget name="properties_0_button"/>
    </widgets>
  </object>
</interface>
